frontend/ui/widget/menu.lua

```diff
@@ -813,6 +813,9 @@ function Menu:init()
         self.key_events.PrevPage = {
             {Input.group.PgBack}, doc = "goto previous page of the menu"
         }
+    end
+
+    if Device:hasDPad() then
```

Comment:
Doesn't this affect regular computer keyboards too? (AppImage)

Reply:
Oh never mind, the SDL device already hasDPad = yes.

Reply:
Yeah, I double-checked that stuff with a keyboard always had hasDPad enabled ;).